frankenphp.go

```diff
@@ -122,6 +122,9 @@ type FrankenPHPContext struct {
 	// Whether the request is already closed by us
 	closed sync.Once
 
+	// whether the context is ready to receive requests
```

Comment:
yeah. basically. This becomes `true` after the worker has called `frankenphp_handle_request` for the first time and `false` upon restart.

The tricky bit here is that we have a WG that is waiting for the worker to call `frankenphp_handle_request`, but it might never happen (e.g., a crash). In this case, we don't want to increment our WG on restart; so we need some way to track state.

Reply:
So maybe "whether the *thread*..." could be more explicit?